draft-irtf-cfrg-randomness-improvements.md

```diff
@@ -197,8 +203,14 @@ Sig MUST be a deterministic signature function, e.g., deterministic ECDSA {{RFC6
 or use an independent (and completely reliable) entropy source, e.g., if Sig is implemented 
 in an HSM with its own internal trusted entropy source for signature generation.
 
-In systems where signature computations are expensive, G'(n) may be precomputed and pooled.
-This is possible since the construction depends solely upon the CSPRNG output and private key.
+In systems where signature computations are expensive, Sig(sk, tag1) may be cached. 
+Experimental results indicate that caching the signature output is critical for performance -- 
```

Comment:
I suggest we remove this sentence and address it in a separate PR. Other than that, this is good!